cmd/mimir/config-descriptor.json

```diff
@@ -5195,6 +5195,36 @@
           "fieldFlag": "alertmanager.max-alerts-size-bytes",
           "fieldType": "int"
         },
+        {
+          "kind": "field",
+          "name": "alertmanager_notify_hook_url",
+          "required": false,
+          "desc": "URL of a hook to invoke before a notification is sent. empty = no hook.",
+          "fieldValue": null,
+          "fieldDefaultValue": "",
+          "fieldFlag": "alertmanager.notify-hook-url",
+          "fieldType": "string"
+        },
+        {
+          "kind": "field",
+          "name": "alertmanager_notify_hook_receivers",
+          "required": false,
+          "desc": "List of receivers to enable notify hooks for. empty = all receivers.",
+          "fieldValue": null,
+          "fieldDefaultValue": "",
+          "fieldFlag": "alertmanager.notify-hook-receivers",
+          "fieldType": "string"
+        },
+        {
+          "kind": "field",
+          "name": "alertmanager_notify_hook_timeout",
+          "required": false,
+          "desc": "Maximum amount of time to wait for a hook to complete before. 0 = no timeout.",
```

Comment:
Is it correct, or should it be "for a hook to complete before timing out"?

Reply:
Good call out, I think this should be updated.